### DISCUSSION TITLE
Is it possible to detect pinch gesture with one hand 2 fingers or two hands each with 2 fingers

### DISCUSSION THREAD

**Discussion by shunmian** [vladmandic/human#441] [Q&A]
Hi, great work on the repo! Just curious if it is possible to detect pinch gesture with one hand 2 fingers or two hands each with 2 fingers?

**Comment by vladmandic** [comment]
possible? sure. you'd need to define what that gesture is, its not built-in.
for example, you'd need to define your gesture like in <https://github.com/vladmandic/human/blob/main/src/hand/fingergesture.ts> and then regsister it

**Comment by vladmandic** [comment]
anyhow, closing this issue and transferring it to discussions as its not a product issue but a general question.
